.github/workflows/tox.yml

```diff
@@ -92,8 +92,9 @@ jobs:
     name: >-
       ${{ matrix.tox_env }}@${{ matrix.os }}
     runs-on: ${{ matrix.os }}
+    needs: linters
```

Comment:
This is not necessary, let everything run in parallel, this is the most responsive way + let's GHA platform schedule the jobs better.

Reply:
Besides, linter failures does not mean that the functional changes cannot be tested/useful without perfect linter results.